SourceSVN/SourceSVN.php

```diff
@@ -253,16 +253,21 @@ public function update_config() {
 	public function commit( $p_repo, $p_data ) {
 		if ( preg_match( '/(\d+)/', $p_data, $p_matches ) ) {
 
+			// Detect if there is a svn:log revprop change, assume not
+			$t_revprop = gpc_get_bool( 'revprop', false );
+
 			$t_url = $p_repo->url;
 			$t_revision = $p_matches[1];
 			$t_svnlog_xml = $this->svn_run( "log -v $t_url -r$t_revision --xml", $p_repo );
 
-			if ( SourceChangeset::exists( $p_repo->id, $t_revision ) ) {
-				echo "Revision $t_revision already committed!\n";
-				return null;
+			if ( $t_revprop == false ) {
```

Comment:
Nitpick: when testing booleans, there is no need to use a comparison, the boolean operator is enough (i.e. `!$t_revprop`).

Reply:
I was following John's coding [a little idiosyncratic at times]. I personally prefer the `if($t_revprop)/if(!$t_revprop)` syntax, so will "correct" this with a new commit.